### DISCUSSION TITLE
Welcome to WSL Discussions!

### DISCUSSION THREAD

**Discussion by craigloewen-msft** [microsoft/WSL#5854] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

The goal of this discussion board is to share surrounding knowledge and topics about WSL. These discussion boards will be more community driven than the regular issues board, and may have less presence from the WSL engineering team. As always if you are experiencing any technical issues or have feature requests please file them [here on our Issues page](https://github.com/microsoft/WSL/issues/new/choose), and any updates from the engineering team will be found there. 

Please feel free to add any thoughts, comments, or questions below.

Thank you!
-The WSL team

**Comment by eduardoarandah** [comment]
**Clipboard integration is my biggest issue**

I know I can select to copy and Ctrl-insert to paste. 
I know there’s some tricks in vim to integrate via some mappings. 
I know they’re two different environments... 
or that i can use vscode remote... 
But still **feels** like a small rock in the shoe. 

Having a shared clipboard would be a good quality of life improvement given that it’s an operation so common in development.

**Reply by WSLUser** [reply]
I feel like this would do more harm than good from a security perspective. Just use interop to use clip.exe in WSL.

**Reply by craigloewen-msft** [reply]
You've identified practically all of our suggested work arounds, and you're right this is a great feature request. :)

**Reply by paulstelian97** [reply]
Uh, shared clipboard with _what_? WSL has no GUI/Xserver or anything...

**Reply by crramirez** [reply]
But, with an Xserver you have a shared clipboard for GUI apps and without GUI besides the clipboard of the terminal that is shared... which clipboard do you want to be shared?

Please explain your use case

**Comment by Chiramisudo** [comment]
Hello WSL! I've been using WSL pretty much since it came out and was immediately in love with the idea. With WSL2, it's become the near-perfect marriage of Linux and Windows. This helps immensely with my workflows in both Git and managing our Linux servers over SSH. I tell other developers about it all the time and encourage them to try it. Long live WSL!!! 😁

**Reply by paulstelian97** [reply]
The CUDA support (coming soon, it's in the previews) further makes me less likely to need an actual dual boot (and is a bonus over normal VMs except if you can PCI-passthrough your NVidia GPU)

**Reply by WSLUser** [reply]
Technically Hyper-V should be getting the same support as the kernel code is for hyper-v. It just happens to also be utilized by WSL2, which is a light-weight version of Hyper-V so it uses the same kernel options.

**Comment by baijnath-sag** [comment]
Hello WSL, from few days i have been trying to install an application on WSL2 with HA enabled. For this i need to start the application executable on 2 different instances of WSL, say one ubuntu and another on alpine. Since i have n number of WSL instances this seems feasible, but i am getting roadblock around the listener being shared. Our application run on 5555 and there is no configuration to change this port, thus i need to live with this port; **when i am running the application on ubuntu, 5555 port is getting acquired and when i tried to run the same application on Alpine i am getting error bind: address already in use**.

Do we have any configuration via which we can disable the service/listener sharing among WSL instances?

**Comment by NotTheDr01ds** [comment]
@baijnath-sag Hello and welcome.  Probably best to start a [*new* discussion](https://github.com/microsoft/WSL/discussions/new) on the topic.  You are just replying to a very old post that introduced the Discussions feature.

You also might want to see [this Super User question](https://superuser.com/q/1715273/1210833) where someone asked a similar question.  Or you might have already seen it -- I just got an upvote on it this morning, and that's rare for that topic, so that might have been you ;-).

**Reply by baijnath-sag** [reply]
Thanks @NotTheDr01ds, will go through the Super User discussion.
